src/main/java/com/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,34 @@
+package com.bookstore.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+public class BookDto {
+    private Long id;
+
+    private String title;
+
+    private String author;
+
+    private String isbn;
+
+    private BigDecimal price;
+
+    private String description;
```

Comment:
don't leave so many empty blank lines, hard to read

Reply:
<img width="1681" alt="Screenshot 2024-10-12 at 14 12 45" src="https://github.com/user-attachments/assets/2aaf5722-ccfc-4bbb-ac0f-50b841c84802">
Initially I did it without indentation - an error appeared